public/docs/ts/latest/quickstart.jade

```diff
@@ -1,381 +1,42 @@
+  Just open it and follow along with the documentation.
+  If you'd rather develop in a local environment, learn how on the [Setup](guide/setup.html) page.
 
+  Angular applications are made of _components_. 
+  A _component_ is the combination of an HTML template and a component class that controls a portion of the screen.
+  Here's a super-simple Angular component, written in [TypeScript](#typescript):
 
++makeExample('app/app.component.ts','','app/app.component.ts')(format='.')
 :marked
-  ### Add some style
-
-  Styles aren't essential, but they're nice, and `index.html` assumes that you have
-  a stylesheet called `styles.css`.
-
-  Create a `styles.css` file in the *#{_indexHtmlDir}* folder, and start styling,
-  perhaps with the minimal styles shown below.
-
-+makeExcerpt('styles.css (excerpt)', 'quickstart')
-
-.callout.is-helpful
-  :marked
-    For the full set of master styles used by the documentation samples,
-    see [styles.css](https://github.com/angular/angular.io/blob/master/public/docs/_examples/_boilerplate/styles.css).
-
-.l-main-section#build-and-run
-h2 Step !{step++}: Build and run the application
-block run-app
-  :marked
-    Open a terminal window and enter this command:
-  code-example.code-shell.
-    npm start
-  aside.is-right
-    :marked
-      [Read more](https://github.com/angular/quickstart/blob/master/README.md#npm-scripts) about
-      other useful npm scripts included in this example's `package.json`.
-
-  :marked
-    That command runs the following two parallel node processes:
-    * The TypeScript compiler in watch mode.
-    * A static file server called _lite-server_ that loads `index.html` in a browser
-    and refreshes the browser when application files change.
-
-    In a few moments, a browser tab should open and display the following:
-
-figure.image-display
-  img(src='/resources/images/devguide/quickstart/hello-angular.png' alt="Output of QuickStart app")
-
-
-block build-app
-  //- Nothing for ts.
-
-.l-main-section#make-changes
-h2 Step !{step++}: Make some live changes
+  The `@Component` [_decorator_](glossary.html#decorator '"Decorator" explained') function takes a _metadata_ object that 
+  describes how the HTML template and component class work together.
+  
+  The `selector` property tells Angular to display the component inside a custom `<my-app>` tag in the `index.html`.
++makeExample('index.1.html','my-app','index.html (body)')(format='.')
 :marked
-  Try changing the message in <span ngio-ex>app/app.component.ts</span> to "Hello Again Angular!".
-
-block server-watching
+  The `template` property defines a message inside an `<h1>` header.
+  The message starts with "Hello" and ends with `{{name}}` 
+  which is an Angular [interpolation binding](guide/displaying-data.html) expression.
+  At runtime, Angular replaces `{{name}}` with the value of the component's `name` property. 
+  Change the component class's `name` property from `'Angular'` to `'World'` and see what happens.
```

Comment:
Consider putting this sentence in a separate paragraph (so that the "Change ..." will stick out). Or maybe make actually make it a call-out?